conda/meta.yaml

```diff
@@ -1,20 +1,21 @@
 {% set name = "jina" %}
-{% set version = "2.0.19" %}
+{% set version = "2.0.23" %}
```

Comment:
this needs to be changed on every release right? could you add it in `scripts/release.sh` L45-49

Reply:
Will do